### DISCUSSION TITLE
Notable differences in the two OS

### DISCUSSION THREAD

**Discussion by ricardfs** [zepp-health/zeppos-forum#13] [Developer Feedback]
Hello.

I am creating watch faces for my wife's Amazfit Bip U Pro model watch, I have found too many differences in the Zepp Creator between devices that work with Zepp OS and those that don't.

For example, in the design screens of the device spheres with Zepp OS there are two elements that do not appear in the design screens of devices that do not work with Zepp OS, instead I have seen and handled spheres of devices that do not work with Zepp OS, for example the Bip U Pro, that do have visual and tactile options implemented that are created with those elements.

For me it is very important to be able to create spheres with icons that when clicking on them take you to the screen of the element, I think that this is achieved with the option to go to the hot zone but in the non-Zepp OS section I do not know you can upload an image to create the visible icon, in the same way the icon element only appears on the design screen of Zepp Creator, once the QR code is used to transfer the sphere the icon does not appear.

Is it possible that it is just an error of the developers of Zepp Creator?

Thank you very much.

**Comment by ricardfs** [comment]
I give the answer to my own question I don't know how but the element to go to the hot zone if it works for me, surely I was doing something wrong.

but the component that can be edited does not exist.

**Reply by sunnyBob** [reply]
1. Set hot zone size
2. Set hot zone position 
3. you could upload a transparent png image if you do not want the image shows in screen
4. click the hot zone you set, will jump to the app page

<img width="1919" alt="image" src="https://user-images.githubusercontent.com/18245566/203899982-ebeeff8a-0902-496c-9eca-554d08abfe6f.png">
